bescort.lic

```diff
@@ -265,6 +283,51 @@ class Bescort
 
   private
 
+  def brocket_1(mode)
+    move 'climb fence' if mode.include?('enter')
```

Comment:
❓ Should this do a check that you're in the right room to start?

Reply:
I can add that, sure.